pydatastructs/trees/tests/test_binary_trees.py

```diff
@@ -68,11 +69,13 @@ def test_BinarySearchTree():
     b.insert(-7, 7)
     b.insert(-14, 14)
     b.insert(-13, 13)
-    assert b.delete(-13) is True
-    assert b.delete(-10) is True
-    assert b.delete(-3) is True
-    assert b.delete(-13) is None
-    bl = BST()
+
+    b.delete(-13)
+    b.delete(-10)
+    b.delete(-3)
+    b.delete(-13)
```

Comment:
Why these `assert` are removed? You said something about it in chat.

Reply:
The correct return value is True (as delete operation happens). It works correctly on my local (which has Python 3.8), but only for the Python 3.8 CI, return value is None. I have opened up an issue for this: https://github.com/codezonediitj/pydatastructs/issues/558